README.md

```diff
@@ -27,7 +27,42 @@ Using this package requires a working Go environment. [See the install instructi
 
 Go Modules are required when using this package. [See the go blog guide on using Go Modules](https://blog.golang.org/using-go-modules).
 
-### Using `v2` releases
+### Golang version 1.14 
+In Go 1.14, modules were made "production-ready" (see [wiki](https:/github.com/golang/go/wiki/Modules#modules)).
+Dependencies for projects were streamlined and there is no need to manually install
+go packages anymore, the go toolchain handles it alone.
+
+To use `cli` in your project :
+
+- If your project is not a module yet, initialize your project with 
+```bash
+go mod init example.com/user/projectName
+```
+(`example.com/user/projectName` is your module name)
+- In the files using `cli`, use this import statement :
+```go
+...
+import (
+  "github.com/urfave/cli/v2" // imports as package "cli"
+)
+...
+```
+Then, install your project module :
+```bash
+go install example.com/user/projectName
+```
+(where `example.com/user/projectName` is your module name)
+This should output:
+```bash
+go: finding module for package github.com/urfave/cli/v2
+go: found github.com/urfave/cli/v2 in github.com/urfave/cli/v2 v2.2.0
+```
+The module has been installed and is ready to use anywhere on your system
+
+For troubleshooting, this [golang doc](https://golang.org/doc/code.html) is useful
+
+
+### Using `v2` releases with older go versions
```

Comment:
This statement is not accurate. The instructions are not different between 1.14 and earlier versions. 

Turns out the name `GO111MODULE` is a bit misleading—it forces modules to be enabled, even in Go versions that do not enable them by default. See `go help modules` for more information, or the release notes for 1.14 to see what's changed: https://golang.org/doc/go1.14

Reply:
The thing is, if your are using Go 1.14 the instructions simply don't work, using the GO111MODULE is sort of a band-aid from what I understood, and not the proper way to install modules at least for 1.14. So even though my PR might be a little too documented , it's better than giving a temporary solution that does not even work for 1.14.

Reply:
From `go help modules`:

> For more fine-grained control, the go command continues to respect
a temporary environment variable, GO111MODULE, which can be set to one
of three string values: off, on, or auto (the default).
If GO111MODULE=on, then the go command requires the use of modules,
never consulting GOPATH. We refer to this as the command
being module-aware or running in "module-aware mode".
If GO111MODULE=off, then the go command never uses
module support. Instead it looks in vendor directories and GOPATH
to find dependencies; we now refer to this as "GOPATH mode."
If GO111MODULE=auto or is unset, then the go command enables or disables
module support based on the current directory.
Module support is enabled only when the current directory contains a
go.mod file or is below a directory containing a go.mod file.

From my understanding, setting `GO111MODULE=on` doesn't enable modules for a project, it just requires the use of modules when running the `go` command. If a project isn't using modules already, you're  right that it probably won't work. But I think we cover that with the line immediately before this section:

> Go Modules are required when using this package. [See the go blog guide on using Go Modules](https://blog.golang.org/using-go-modules).

That said, I'm not entirely clear on what worked in Go 1.13 that no longer works in Go 1.14. If you could walk me through your local setup or point me to the part of the 1.14 docs that cover the difference, that might be helpful.